frontend/src/CustomMarker/CustomMarker.tsx

```diff
@@ -1,31 +1,31 @@
 import React from "react";
```

Comment:
accidental changes I think; can this file be removed from the PR?

Reply:
I think this was done by my Prettify somehow.... I'll copy the file back from main lol